opendm/config.py

```diff
@@ -17,28 +31,34 @@ def __call__(self, parser, namespace, values, option_string=None):
 
 def config():
     parser.add_argument('--images', '-i',
-                        metavar='<string>',
+                        metavar='<path>',
                         help='Path to input images'),
 
     parser.add_argument('--project-path',
-                        metavar='<string>',
-                        help='Path to the project to process')
+                        metavar='<path>',
+                        help='Path to the project folder',
+                        default=defaultSettings['project_path'])
+
+    parser.add_argument('name',
```

Comment:
Is this parameter being added to allow a user to have:

/project/a/images
/project/b/images
...

And thus call ODM with: `run.py --project-path /project/ --name a`?

Or what is the intended use?